tests/test_persisted_task_properties.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for validating persisted task properties for different data types."""
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+
+
+class TestPersistedTaskPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for
+    persisted task properties for different data types.
+    """
+
+    @pytest.fixture(scope="class")
+    def voltage_tester_task(self):
+        """Gets the voltage tester task from the persisted tasks in the system."""
+        system = nidaqmx.system.System.local()
+        voltage_tester_task = system.tasks[0]
```

Comment:
this isn't very safe. if someone changes the imports it might break. the `PersistedTaskCollection` allows for indexing by string, so you can ask for it by name

Reply:
Updated to take the persisted task by name